docs/guides/getting_started/terminology.md

```diff
@@ -15,11 +15,15 @@ to stay in line with Discord internally.
 
 Discord.Net is split into a core library and two different
 implementations - `Discord.Net.Core`, `Discord.Net.Rest`, and
-`Discord.Net.WebSockets`.
+`Discord.Net.WebSocket`.
 
-As a bot developer, you will only need to use `Discord.Net.WebSockets`,
+As a common bot developer, you will only need to use `Discord.Net.WebSocket`,
```

Comment:
What do you mean by a "common bot developer" as opposed to just a "bot developer"?

Reply:
Developers who implement the REST interactions system [do not use websocket](https://github.com/Rozen4334/Barriot/blob/master/Barriot.Core/Program.cs#L25) at all. Because it is so uncommon for people to use this system in dnet, I chose to mention that you commonly do not need it.

Reply:
Alright, that makes sense. Perhaps this sounds better to remove the need to clarify `common`?
```suggestion
You will typically only need to use `Discord.Net.WebSocket`,
```